.github/workflows/integration-test.yml

```diff
@@ -47,36 +47,6 @@ jobs:
           all_but_latest: true # can cancel workflows scheduled later
 
   test-integration:
-    name: Integration - Linux, ${{ matrix.test }}
```

Comment:
Is this the last integration test?!

Reply:
I actually still see `humio` here.

Reply:
Nvm, I see that one was migrated in #10815